components/lib/treeselect/treeselect.d.ts

```diff
@@ -87,6 +94,7 @@ export interface TreeSelectProps extends Omit<React.DetailedHTMLProps<React.Inpu
     panelClassName?: string;
     appendTo?: TreeSelectAppendToType;
     emptyMessage?: string;
+    expandedKeys?: TreeSelectExpandedKeysType;
```

Comment:
Please update `\api-generator\components\treeselect.js` with the new `expandedKeys` property as well.